.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: rust
+rust:
+- stable
+- beta
+- nightly
+sudo: false
+install:
+- wget https://github.com/xiph/speexdsp/releases/download/SpeexDSP-1.2rc3.tar.gz
+- tar xvfz SpeexDSP-1.2rc3.tar.gz
+- cd SpeexDSP-1.2rc3 && ./configure --prefix=$HOME/installed_speedxsp && make && make install &&
+  cd ..
+- export PKG_CONFIG_PATH=$HOME/installed_speedxsp/lib/pkgconfig:$PKG_CONFIG_PATH
+- export LD_LIBRARY_PATH=$HOME/installed_speedxsp/lib:$LD_LIBRARY_PATH
+script:
+- cargo build --verbose
```

Comment:
Without `--features sys` it just builds the rust transpiled version. Maybe it should test both?

Reply:
Yes sure, thought they were no features but just checked. Will make the changes

Reply:
We could just use the default and avoid the whole speexdsp C library alternatively :)

Reply:
What do you mean by "avoid the whole library"? :stuck_out_tongue:

Reply:
`libspeexdsp` is not needed by default, the `sys` feature would pull it in optionally.